# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1458588908 14400
#      Mon Mar 21 15:35:08 2016 -0400
# Branch stable
# Node ID 5162cdfb56d8e31ad00dc260a26d1c92c69c2acf
# Parent  04e0598f8b94f89e3bf1e4f6dd5c3d3503e350f6
Version 4.0.1 released.

* configure.ac (OCTAVE_VERSION): Now 4.0.1.
(OCTAVE_PATCH_VERSION): Now 1.
(OCTAVE_COPYRIGHT): Update year to 2016.
(OCTAVE_RELEASE_DATE): Set to 2016-03-21.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,31 +14,31 @@ dnl Copyright (C) 1993-2015 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.63])
-AC_INIT([GNU Octave], [4.0.1-rc4], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.0.1], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=0
-OCTAVE_PATCH_VERSION=1-rc4
+OCTAVE_PATCH_VERSION=1
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
-OCTAVE_COPYRIGHT="Copyright (C) 2015 John W. Eaton and others."
-
-OCTAVE_RELEASE_DATE="2016-01-18"
+OCTAVE_COPYRIGHT="Copyright (C) 2016 John W. Eaton and others."
+
+OCTAVE_RELEASE_DATE="2016-03-21"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do that is
 ## with library versioning, but not all systems support that.
 ## NOTE: This macro will be removed in a future version of Octave.  If
 ## you insist on checking for features using a version number, use the
 ## OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, and
